x-pack/plugins/synthetics/common/constants/monitor_defaults.ts

```diff
@@ -37,6 +37,7 @@ export const DEFAULT_COMMON_FIELDS: CommonFields = {
     unit: ScheduleUnit.MINUTES,
   },
   [ConfigKey.APM_SERVICE_NAME]: '',
+  [ConfigKey.CONFIG_ID]: '',
```

Comment:
Do we need a migration to map this field to monitor id for all monitors?

Reply:
i don't think so. It can be done when we decide to migrate existing integrations.